### ISSUE TITLE
Please provide a man page for obs-cli

### ISSUE THREAD

**Issue by bdrung** [muesli/obs-cli#27]
I am packaging obs-cli for Debian (see [Debian bug #1000489](https://bugs.debian.org/1000489)). According to Debian, each binary in /usr/bin should have a manual page. See https://lintian.debian.org/tags/no-manual-page for more reasoning.

Please provide a man page for obs-cli. The man page should list which command and subcommand exists and what parameters they take. You can write the man page in Markdown and convert it to `nroff`. See https://github.com/bdrung/ionit/blob/master/ionit.1.md for an example.

**Comment by muesli** [comment]
Indeed, we should provide one. Since we're using Cobra, we can use it to automatically generate a man page from the command structure within obs-cli.

**Comment by muesli** [comment]
See progress in #33.
